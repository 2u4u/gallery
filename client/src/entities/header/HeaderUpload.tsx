type Props = {
    handleOpenModal: () => void;
}

function HeaderUpload({ handleOpenModal }: Props) {

    return (
        <button onClick={handleOpenModal} className='header__upload upload'>
            <img src="./img/upload.svg" alt="upload file" className='upload__img' />
            Upload file
        </button>
    );
}

export default HeaderUpload;
