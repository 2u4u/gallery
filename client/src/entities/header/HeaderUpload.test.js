import renderer from 'react-test-renderer';
import HeaderUpload from './HeaderUpload';

describe('Header upload component ', () => {
    it('matches snapshot', () => {
        const handleOpenModal = () => false;
        const component = renderer.create(
            <HeaderUpload 
                handleOpenModal={handleOpenModal}
            />,
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
