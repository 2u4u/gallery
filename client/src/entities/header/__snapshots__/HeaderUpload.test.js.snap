// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header upload component  matches snapshot 1`] = `
<button
  className="header__upload upload"
  onClick={[Function]}
>
  <img
    alt="upload file"
    className="upload__img"
    src="./img/upload.svg"
  />
  Upload file
</button>
`;
